### PR TITLE
add gitlab flow

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 This repository's `master` branch tracks images pushed to the official Demisto Docker Hub organization at: https://hub.docker.com/u/demisto/. Other branches` images are pushed to [devdemisto](https://hub.docker.com/u/devdemisto).
 
 **Note:** We generate nightly information about packages and os dependencies used in each of Demisto's docker images. Checkout the `dockerfiles-info` project [README](https://github.com/demisto/dockerfiles-info/blob/master/README.md) for a full listing.
-
+Test
```
Comment (JudahSchwartz) [inline README.md:8]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -5,7 +5,6 @@
 This repository's `master` branch tracks images pushed to the official Demisto Docker Hub organization at: https://hub.docker.com/u/demisto/. Other branches` images are pushed to [devdemisto](https://hub.docker.com/u/devdemisto).
 
 **Note:** We generate nightly information about packages and os dependencies used in each of Demisto's docker images. Checkout the `dockerfiles-info` project [README](https://github.com/demisto/dockerfiles-info/blob/master/README.md) for a full listing.
```
Comment (JudahSchwartz) [inline README.md:7]:
```suggestion
**Note:** We generate nightly information about packages and os dependencies used in each of Demisto's docker images. Checkout the `dockerfiles-info` project [README](https://github.com/demisto/dockerfiles-info/blob/master/README.md) for a full listing.

```

File: docker/build_docker.sh [hunk 3]
```diff
@@ -289,6 +290,8 @@ function docker_build {
     if docker_login; then
         env DOCKER_CONTENT_TRUST=$docker_trust DOCKER_CONFIG="${DOCKER_CONFIG}"  docker push ${image_full_name}
         echo "Done docker push for: ${image_full_name}"
+        PUSHED_DOCKERS="${image_full_name},$PUSHED_DOCKERS"
```
Comment (tkatzir) [inline docker/build_docker.sh:293]: There will be an extra comma at the end. Is that ok?

File: docker/build_docker.sh [hunk 4]
```diff
@@ -418,3 +422,5 @@ for docker_dir in `find $SCRIPT_DIR -maxdepth 1 -mindepth 1 -type  d -print | so
         echo ">>>>>>>>>>>>>>> `date`: Done docker build <<<<<<<<<<<<<"
     fi
 done
+echo "Successfully pushed $PUSHED_DOCKERS"
+echo $PUSHED_DOCKERS > $CIRCLE_ARTIFACTS/pushed_dockers.txt
```
Comment (tkatzir) [inline docker/build_docker.sh:426]: Do the `echo` after file creation, or add another echo so we know the file was created successfully.

**Review by tkatzir — APPROVED** [review]
Great!
Approved, with comments.

File: docker/build_docker.sh (hunk 3 above)
Comment (JudahSchwartz) [reply docker/build_docker.sh:293]: yeah it actually gets parsed with `sed 's/,$//')` in the end to drop the trailing comma downstream. Let me know if thats too ugly =)

Comment (tkatzir) [reply docker/build_docker.sh:293]: No, it's ok. Leave it as is.

**Pull request merged**